### PR TITLE
organizations: display stats about apps installed in clusters

### PR DESCRIPTION
**Pull request by axbarsan**
This PR adds stats about apps that are installed in the clusters that belong to the selected organization:
* `Apps in use` -> the total number of apps installed in the cluster (if an app is installed multiple times, it's only counted once)
* `App deployments` -> the total number of apps installed in the cluster (also counting apps installed more than once)

The PR adds a couple very happy path integration tests for the whole stats thingy, and a few fixes for the K8s API types.

Don't get discouraged by the PR size, there are 12312958 mock responses for K8s API requests.

### Preview

image

### CODE REVIEW

File: __tests__/ClusterManagement.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ describe('ClusterManagement', () => {
 
   /************ TESTS ************/
   it('creates a v5 cluster and redirect to details view', async () => {
+    jest.setTimeout(20000);
```
Comment (axbarsan) [inline __tests__/ClusterManagement.js:50]: This is a flaky test that fails 2/5 times. I increased its timeout so it won't bother us, for now

**Comment by marians** [comment]
FYI @cokiengchiara - This adds a little bit of app usage summary to the new organization details page for SSO/Management API users.

**Review by marians — APPROVED** [review]
Tested and LGTM

File: src/components/MAPI/organizations/OrganizationDetailGeneral/utils.ts [hunk 2]
```diff
@@ -410,3 +416,64 @@ export function fetchReleasesSummaryKey(
...
+ * @param clusters
+ */
+export function fetchAppsSummaryKey(
+  clusters?: capiv1alpha3.ICluster[]
+): string | null {
+  if (!clusters) return null;
+
+  const clusterKeys = clusters.map(
+    (c) => `${c.metadata.namespace}/${c.metadata.name}`
+  );
```
Comment (kopiczko) [inline src/components/MAPI/organizations/OrganizationDetailGeneral/utils.ts:476]: Shouldn't clusters be sorted first?

Comment (axbarsan) [reply src/components/MAPI/organizations/OrganizationDetailGeneral/utils.ts:476]: That would be the intelligent thing to do 😬

**Pull request merged**